commit 0567a6337b84fa045b5732e98203f488274aa2a2
Author: Bram Verweij <amverweij@gmail.com>
Date:   Wed Nov 8 18:00:52 2006 +0200

    xfree86/linux acpi: fix tokenising
    Split on a space, rather on the 'video' string, as strtok takes a char,
    not a string.

diff --git a/hw/xfree86/os-support/linux/lnx_acpi.c b/hw/xfree86/os-support/linux/lnx_acpi.c
index aa30e72..024e6ef 100644
--- a/hw/xfree86/os-support/linux/lnx_acpi.c
+++ b/hw/xfree86/os-support/linux/lnx_acpi.c
@@ -78,7 +78,7 @@ lnxACPIGetEventFromOs(int fd, pmEvent *e
 	char *data = NULL; /* doesn't appear to be used in the kernel */
 	unsigned long int notify_l, data_l;
 
-	video = strtok(ev, "video");
+	video = strtok(ev, " ");
 
 	GFX = strtok(NULL, " ");
 #if 0
